### DISCUSSION TITLE
How to use additional Clarity classes

### DISCUSSION THREAD

**Discussion by okelet** [chipzoller/hugo-clarity#106] [Q&A]
Hi! The documentation says that the theme is based on the [Clarity Design System](https://clarity.design/). I am trying to create a shortcode for alerts (https://clarity.design/documentation/alerts), but I don't get any markup. Am I missing something? Thanks!

This is the shortcode (`layouts/shortcodes/alert.html`):

```html
<div class="alert alert-{{ .Get 0 }}" role="alert">
  <div class="alert-items">
      <div class="alert-item static">
          <div class="alert-icon-wrapper">
              <clr-icon class="alert-icon" shape="check-circle"></clr-icon>
          </div>
          <span class="alert-text">{{ .Inner | markdownify | emojify }}</span>
      </div>
  </div>
</div>
```

And this is how I use it:

```
{{% alert info %}}
My text :)
{{% /alert %}}
```

**Comment by onweru** [comment]
@okelet, we based the theme on clarity design principles. Not clarity’s css libraries
